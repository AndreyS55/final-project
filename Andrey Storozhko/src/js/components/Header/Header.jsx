import React from 'react';
import Logo from '../Logo/Logo';
import NavBar from '../NavBar/NavBar';

import styles from './Header.scss';

const Header = () => (
  <header className={styles.pokedex__header}>
    <Logo />
    <NavBar />
  </header>
);

export default Header;