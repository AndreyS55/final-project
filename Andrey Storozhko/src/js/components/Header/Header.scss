.pokedex__header {
  display: flex;
  height: 70px;
  background-color: #0074D9;
  justify-content: space-between;
  align-items: center;
}