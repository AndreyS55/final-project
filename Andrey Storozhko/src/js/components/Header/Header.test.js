import React from 'react';
import Header from './Header';

it('should render a Header', () => {
  const component = shallow(<Header />);
  expect(component).toMatchSnapshot();
});
