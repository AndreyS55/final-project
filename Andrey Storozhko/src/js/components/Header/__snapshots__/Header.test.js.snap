// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render a Header 1`] = `
<header
  className="pokedexHeader"
>
  <div
    className="headerWrapper"
  >
    <Logo />
    <NavBar />
  </div>
</header>
`;
